GraphBulkImporter/Extensions.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Microsoft.Azure.CosmosDB.BulkExecutor.Graph.Element;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphBulkImporter
+{
+    public static class Extension
+    {
+        /// <summary>
+        /// Turns an object in to a GremlinVertex with id & label, and a property bag of all other properties
+        /// If obj does not have a property named id and partitionKey property, or you want a custom label applied 
+        /// use an overload that allows you to specify which property will be used as "id" and "partitionKey"
+        /// as well as allowing you to set a custom label
+        /// </summary>
+        /// 
+        /// <returns>
+        /// A new GremlinVertex object 
+        /// with its id and partitionKey properties set to the corresspinding id and partitionKey properties of the object being converted
+        /// and the label property set to the type name of the object
+        /// </returns>
+        public static GremlinVertex ToGremlinVertex(this object obj)
+        {
+            var label = obj.GetType().Name;
+            
+            if (!obj.HasProperty("id"))
```

Comment:
should make these const fields since they are used in multiple places, safer from typos